questdb-rs/src/ingress/mod.rs

```diff
@@ -1822,6 +2052,19 @@ impl SenderBuilder {
                 "retry_timeout" => {
                     builder.retry_timeout(Duration::from_millis(parse_conf_value(key, val)?))?
                 }
+
+                #[cfg(feature = "ilp-over-http")]
+                "disable_line_protocol_validation" => {
```

Comment:
Let's rename this to something better like "autodetect_protocol" where `true` is default.

Reply:
This way if a user wants to override they just specify:
```protocol_version=2``` and that's it.

Reply:
Currently, `protocol_version=auto` cannot replace the `disable_line_protocol_validation` configuration. Regardless of the value configured for `protocol_version`, the sender will request the server to verify whether the client's configuration is within the server's support range. The role of `disable_line_protocol_validation` is to explicitly turn it off if the user does not want the verification process(a way to trun off the round trip for permance.).